File: 백준/Bronze/2712. 미국 스타일/미국 스타일.cs
 int n = int.Parse(Console.ReadLine());
for (int i = 0; i < n; i++)
{
    string[] input = Console.ReadLine().Split();
    double a = double.Parse(input[0]);
    string tp = input[1];

    if (tp == "kg")
        Console.WriteLine($"{a * 2.2046:F4} lb");
    else if (tp == "g")
        Console.WriteLine($"{a * 3.7854:F4} l");
    else if (tp == "l")
        Console.WriteLine($"{a * 0.2642:F4} g");
    else if (tp == "lb")
        Console.WriteLine($"{a * 0.4536:F4} kg");
}

File: 백준/Bronze/2712. 미국 스타일/미국 스타일.py
n = int(input())

for _ in range(n):
    a, tp = input().split()
    a = float(a)

    if tp == "kg":
        print(f"{a * 2.2046:.4f} lb")
    elif tp == "g":
        print(f"{a * 3.7854:.4f} l")
    elif tp == "l":
        print(f"{a * 0.2642:.4f} g")
    elif tp == "lb":
        print(f"{a * 0.4536:.4f} kg")
